index.html

```diff
@@ -8,129 +8,34 @@
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
-    <title>Benjamin Patin's Developer Portfolio</title>
+    <title>Benjamin Patin's Developer Portfolio for Microverse course</title>
   </head>
   <body>
```

Comment:
Page has responsiveness issues on `1024px` and `1440px` as it is aligned to the left. Kindly see the screenshot below.
![laptop](https://user-images.githubusercontent.com/35504149/139495045-3ac1f19e-325a-4a41-9a9f-5344cc0ac258.png)

Reply:
Hey, I am not sure about the assignment, but I based my design based on the figma file of 1440px - it doesn't mention about the 1024px